src/main/java/com/ibm/cloud/sdk/core/security/IamToken.java

```diff
@@ -51,28 +52,51 @@ public Long getExpiration() {
     return expiration;
   }
 
+  IamToken() {
```

Comment:
Is this a constructor?  That takes no arguments? This is really confusing!

Reply:
Yeah, I was adding package-private constructor that's called when we deserialize an IAM response. It does look a bit weird though. I can just make it private.